profiles/integration-test/config.properties

```diff
@@ -0,0 +1,8 @@
+profile=integration-test
+project.version=1.0-SNAPSHOT
+project.url=https://github.com/reiniergs/polyfill-service
+
+path.polyfillsDist=./polyfills/__dist
+path.browserBaselines=${path.polyfillsDist}/browserBaselines.json
+path.browserAliases=${path.polyfillsDist}/browserAliases.json
+path.featureAliases=${path.polyfillsDist}/aliases.json
```

Comment:
what if dev and integration test are active same time, which properties is gonna be set :) ... it's posible tha multiple profiles get active when maven runs